.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,36 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bubkoo/potential-duplicates@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Label to set, when potential duplicates are detected.
+          label: potential duplicate
+          # Exclude keyworlds in title before detecting.
+          exclude: |
+            icon
+            icons
+            logo
+            logos
+            request
+            request:
```

Comment:
Are these needed separately as well? (quite possible if the exclusion is "word" based)

If so, ideally the action would be updated to ignore punctuation before and after a word - or at least have an option for this. What do you think @service-paradis

Reply:
With the current action, it would be needed to have them separated. I can remove `request:`. `request` will still be removed from the comparison string, but it will keep `:`. It could lead to have a little bit more false positive than expected.

I can ask the action author if it would be possible to ignore punctuations.

Reply:
Ah okay, thanks for the explanation - it's a slightly different problem than I expected. I would indeed expect the action to ignore punctuation (or, again, at least have an option to do so).

If the Action's maintainer(s) agree I think we should remove it already now, otherwise I would suggest we ignore most punctuation manualy - am I correct that a including just `:` in the list would remove all colons?

Relatedly, if this works the way you say than I actually think the configuration is currently invalid. The action would first remove the string "request" from the issue text, and then try to remove "request:". But "request:" won't be in the issue text because it just removed "request".

Reply:
> Relatedly, if this works the way you say than I actually think the configuration is currently invalid. The action would first remove the string "request" from the issue text, and then try to remove "request:". But "request:" won't be in the issue text because it just removed "request".

You're right, my bad 🤦‍♂️ 

> am I correct that a including just : in the list would remove all colons?

You're correct. We might be ok if we list potential punctuation marks (other than colon, not much used in issue title) as exclusion.

Reply:
Should we still not exclude "all" punctuation marks. Beyond the template, in an extreme case, if one issue contains punctuation and an "identical" issue is opened that is just missing the punctuation, I think the issues should be considered identical (of course, it is unlikely that punctuation by itself is going to bring the similarity value below the threshold of 0.7, but still...)

Reply:
I just checked again the source of the action. It does, in fact, remove a lot of punctuation and common words (https://github.com/bubkoo/potential-duplicates/blob/master/src/dic.ts).
Colo (`:`) is missing though 😅

Reply:
Can you open an issue or submit a Pull Request to expand that list?

Reply:
Sure, that is on my list :)

Reply:
https://github.com/bubkoo/potential-duplicates/pull/18 is now merged!

Reply:
Sound good, @service-paradis :+1: 

Wondering if this should be updated given the updates to the issue templates from #6303

Reply:
In that case, I think we can give this a try. Perhaps you can ask for a review from newer members to see (1) make them aware and (2) see if they have any objections?